api/src/certification/session/domain/usecases/get-cpf-presigned-urls.js

```diff
@@ -1,19 +1,15 @@
 import { config } from '../../../../../lib/config.js';
-import { S3ObjectStorageProvider } from '../../../../shared/storage/infrastructure/providers/S3ObjectStorageProvider.js';
 
-const { cpf } = config;
+const getPreSignedUrls = async function ({ date, cpfExportsStorage }) {
+  const filesInBucket = await cpfExportsStorage.findAll();
```

Comment:
Commentaire de @mariannebost : on a encore une vision très technique dans ce usecase, on pourrait mettre un vocabulaire + métier à moindre coût

```suggestion
  const cpfExportFiles = await cpfExportsStorage.findAll();
```

Reply:
Done